### PR TITLE
Add a html route provider for entity and entity bundles.

### PR DESCRIPTION
**Pull request by Jaesin**
Adds a route provider for the bundle entity.

### CODE REVIEW

**Comment by Jaesin** [comment]
Because of recent commits, I slimmed this PR down to just adding a route provider for the bundle entity which lets us eliminate the routing file.

File: src/Entity/Routing/EntityTypeHtmlRouteProvider.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  public function getRoutes(EntityTypeInterface $entity_type) {
+    $routes = parent::getRoutes($entity_type);
+
+    $entity_type_id = $entity_type->id();
+
+    if ($collection_route = $this->getCollectionRoute($entity_type)) {
+      $routes->add("entity.{$entity_type_id}.collection", $collection_route);
+    }
+
+    if ($add_form_route = $this->getAddFormRoute($entity_type)) {
```
Comment (dawehner) [inline src/Entity/Routing/EntityTypeHtmlRouteProvider.php]: With https://github.com/Jaesin/content_entity_base/pull/35/files I think we just need the collection one

Comment (dawehner) [inline src/Entity/Routing/EntityTypeHtmlRouteProvider.php]: Oh well, the CruiUIRouteProvider actually has the collection, so i'm not sure why this one is actually needed

**Comment by Jaesin** [comment]
First error is no `entity.ceb_test_content_type.edit_form`.

**Pull request merged**